### PR TITLE
Handle overs with multiple values and multiple arrays.

### PR DESCRIPTION
**Pull request by jonnyh404**
Expands out to multiple OVERs rather than stuffing one over with all the values.

### CODE REVIEW

File: ResSimpy/Nexus/DataModels/StructuredGrid/NexusGrid.py [hunk 1]
```diff
@@ -1458,21 +1458,27 @@ def load_nexus_overs(file_content_as_list: list[str]) -> list[NexusOver]:
...
-                        operator = operator_matches[0]
-                        # remove the operator and the remaining string is the value
-                        value = float(split_line[6][1:])
-                    overs_list.append(NexusOver(arrays=arrays, grid=grid, fault_name=fname,
-                                                i1=i1, i2=i2, j1=j1, j2=j2, k1=k1, k2=k2, operator=operator,
-                                                value=value, threshold=threshold_value))
-                    threshold_value = None
+                    # cut out the ranges
+                    split_line = split_line[6:]
+                    for array in arrays:
```
Comment (cflynn3) [inline ResSimpy/Nexus/DataModels/StructuredGrid/NexusGrid.py:1463]: Where does arrays get assigned a non empty value?

Comment (jonnyh404) [reply ResSimpy/Nexus/DataModels/StructuredGrid/NexusGrid.py:1463]: In the check for OVER token block: 
```
            if nfo.check_token('OVER', line):
                reading_over = True
                over_split_line = nfo.split_line(line)
                arrays = over_split_line[over_split_line.index('OVER')+1:]
```

Comment (cflynn3) [reply ResSimpy/Nexus/DataModels/StructuredGrid/NexusGrid.py:1463]: I see! Please could you put a comment before this line explaining that for when we have to come back to this at a later date.

**Review by cflynn3 — APPROVED** [review]
Changes look good.

**Review by kwr3 — APPROVED** [review]
LGTM

**Pull request merged**